.github/workflows/create-release.yml

```diff
@@ -57,6 +57,7 @@ jobs:
         id: attest
         with:
           subject-path: "dist/*"
+          subject-digest: ${{ steps.push.outputs.digest }}
```

Comment:
Hold on: I don't think that this is going to work without also supplying a `subject-name` (I'm reading the `attest.git` action code at the moment).

Reply:
Additionally, `subject-digest` appears to require `sha256` -- GitHub `git` repositories still tend to (always?) use `sha1`, AFAIK.

Reply:
> Hold on: I don't think that this is going to work without also supplying a `subject-name` (I'm reading the `attest.git` action code at the moment).

It appears that `subject-name` can be inferred from the `subject-path` when the former is absent; so in fact, I think that this is OK as-is.

Reply:
> Additionally, `subject-digest` appears to require `sha256` -- GitHub `git` repositories still tend to (always?) use `sha1`, AFAIK.

This is **not** required when the subject is provided based on a file path (`subject-path`); in those cases, a SHA256 digest of the content will be created when the action runs, and this is separate to the `subject-digest` that can be used to reflect the origin source control commit.  So, cautiously, I also think that this is OK as-is.